rpc_jobs/long_running_slave.yml

```diff
@@ -0,0 +1,102 @@
+- job:
+    name: Long-Running-Slave
+    project-type: workflow
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      # See params.yml
+      - rpc_gating_params
+      - single_use_slave_params:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+          REGION: "IAD"
+          FLAVOR: general1-8
+      - rpc_maas_params:
+          RPC_MAAS_BRANCH: master
+      - string:
+          name: JENKINS_NODE_EXECUTORS
+          default: "10"
```

Comment:
I'll increase it to 15, but the I don't want to match the CIT slave as that is under resourced for the number of executors it has. Also I'm planning to build multiple cloud slaves for redundancy and parallelism.